src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java

```diff
@@ -578,7 +586,8 @@ public double evaluate(final double[] values, final double[] weights,
      * @throws MathIllegalArgumentException if the parameters are not valid
      * @since 2.1
      */
-    public double evaluate(final double[] values, final double[] weights, final double mean)
+    @SuppressWarnings("index:argument.type.incompatible") // values.length is @NonNegative @LTLengthOf(value = {"values", "weights"}, offset = {"0 - 1", "0 - 1"})
```

Comment:
This is slightly different, but related, to the other errors you're getting with just `values`. Can you write a short program that demonstrates both this version of the problem and the version with a single array, and submit a bug? Thanks.